lib/srv/db/cloud/aws.go

```diff
@@ -196,18 +193,20 @@ func (r *awsClient) ensureIAMPolicy(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	if len(placeholders) > 0 {
+		r.log.Warnf("Placeholders %q are found when configuring the IAM policy for database %v. Please make sure the database agent has the IAM permissions to fetch cloud metadata, or make sure these values are set in the static config.",
```

Comment:
I think that the `Placeholders` part is less important that missing IAM permission. Can we swap the sentence and treat the placeholders as a error details nor root cause. Most like the 'placeholders` doesn't mean to much to teleport user.